service/src/test/java/com/bankbazaar/kafka/service/controller/KafkaControllerTest.java

```diff
@@ -46,7 +45,10 @@
     void userController() throws Exception {
         ClassLoader classLoader = getClass().getClassLoader();
 
-        assertEquals(consumeApi("test2.csv"), Status.NULL);
+        /**
+         * controller returns null before any db entry
+         */
+        assertNull(consumeApi("test2.csv"));
```

Comment:
This is a scenario where the given filename does not exist right? Should it not then respond with a 404 status?